DeploymentGuide.md

```diff
@@ -12,25 +12,36 @@ The Management Server can be deployed locally or to the cloud. We have setup a o
 
```

Comment:
How about calling this line: 
### Deploy App via Click-to-Deploy

And then the first sub-step would be:
#### Create an Instance

This mean the CLI section would be:
#### Deploy via Command Line

I think that this change would better reflect what is happening?  How do you think?